templates/enketo/config.json.j2

```diff
@@ -25,7 +25,7 @@
     "themes supported":[{% for theme in enketo_supported_themes %}"{{ theme }}"{% if loop.index != loop.length %}, {% endif %}{% endfor %}],
     "base path": "",
     "log": {
-        "submissions": {{ enketo_enable_submission_logging }}
```

Comment:
Let's add a test for this. Like [this](https://github.com/onaio/ansible-enketo/blob/8cb454b4a25a123651c1f05d07ee0592509427ef/molecule/theme-replacements/verify.yml#L15-L22) one